### ISSUE TITLE
Call to Action

### ISSUE THREAD

**Issue by iamtakashi** [Automattic/blockpatterns#106]
![Pattern 117](https://user-images.githubusercontent.com/908665/90044684-e7921500-dcc5-11ea-8675-6db223476b8d.jpg)

```
<!-- wp:jetpack/layout-grid {"gutterSize":"none","addGutterEnds":false,"column1DesktopSpan":6,"column1TabletSpan":4,"column1MobileSpan":4,"column2DesktopSpan":6,"column2TabletSpan":4,"column2MobileSpan":4,"className":"column1-desktop-grid__span-6 column1-desktop-grid__row-1 column2-desktop-grid__span-6 column2-desktop-grid__start-7 column2-desktop-grid__row-1 column1-tablet-grid__span-4 column1-tablet-grid__row-1 column2-tablet-grid__span-4 column2-tablet-grid__start-5 column2-tablet-grid__row-1 column1-mobile-grid__span-4 column1-mobile-grid__row-1 column2-mobile-grid__span-4 column2-mobile-grid__row-2"} -->
<div class="wp-block-jetpack-layout-grid alignfull column1-desktop-grid__span-6 column1-desktop-grid__row-1 column2-desktop-grid__span-6 column2-desktop-grid__start-7 column2-desktop-grid__row-1 column1-tablet-grid__span-4 column1-tablet-grid__row-1 column2-tablet-grid__span-4 column2-tablet-grid__start-5 column2-tablet-grid__row-1 column1-mobile-grid__span-4 column1-mobile-grid__row-1 column2-mobile-grid__span-4 column2-mobile-grid__row-2 wp-block-jetpack-layout-gutter__nowrap wp-block-jetpack-layout-gutter__none"><!-- wp:jetpack/layout-grid-column -->
<div class="wp-block-jetpack-layout-grid-column wp-block-jetpack-layout-grid__padding-none"><!-- wp:cover {"url":"https://dotcompatterns.files.wordpress.com/2020/08/apostolos-vamvouras-irmseb92la8-unsplash-edit.jpg","id":1883,"dimRatio":0,"minHeight":640} -->
<div class="wp-block-cover" style="background-image:url(https://dotcompatterns.files.wordpress.com/2020/08/apostolos-vamvouras-irmseb92la8-unsplash-edit.jpg);min-height:640px"><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"Write title…","className":"margin-bottom-half","style":{"color":{"text":"#ffffff"},"typography":{"fontSize":29,"lineHeight":"1.24"}}} -->
<p class="has-text-align-center margin-bottom-half has-text-color" style="line-height:1.24;font-size:29px;color:#ffffff">Women</p>
<!-- /wp:paragraph -->

<!-- wp:buttons {"align":"center","className":"margin-top-half"} -->
<div class="wp-block-buttons aligncenter margin-top-half"><!-- wp:button {"borderRadius":0,"style":{"color":{"text":"#ffffff"}},"className":"is-style-outline"} -->
<div class="wp-block-button is-style-outline"><a class="wp-block-button__link has-text-color no-border-radius" style="color:#ffffff">SHOP NOW</a></div>
<!-- /wp:button --></div>
<!-- /wp:buttons --></div></div>
<!-- /wp:cover --></div>
<!-- /wp:jetpack/layout-grid-column -->

<!-- wp:jetpack/layout-grid-column -->
<div class="wp-block-jetpack-layout-grid-column wp-block-jetpack-layout-grid__padding-none"><!-- wp:cover {"url":"https://dotcompatterns.files.wordpress.com/2020/08/altin-ferreira-yzb-df6hz4s-unsplash-edit.jpg","id":1884,"dimRatio":0,"focalPoint":{"x":"1.00","y":"0.50"},"minHeight":640} -->
<div class="wp-block-cover" style="background-image:url(https://dotcompatterns.files.wordpress.com/2020/08/altin-ferreira-yzb-df6hz4s-unsplash-edit.jpg);min-height:640px;background-position:100% 50%"><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"Write title…","className":"margin-bottom-half","style":{"color":{"text":"#ffffff"},"typography":{"fontSize":29,"lineHeight":"1.24"}}} -->
<p class="has-text-align-center margin-bottom-half has-text-color" style="line-height:1.24;font-size:29px;color:#ffffff">Men</p>
<!-- /wp:paragraph -->

<!-- wp:buttons {"align":"center","className":"margin-top-half"} -->
<div class="wp-block-buttons aligncenter margin-top-half"><!-- wp:button {"borderRadius":0,"style":{"color":{"text":"#ffffff"}},"className":"is-style-outline"} -->
<div class="wp-block-button is-style-outline"><a class="wp-block-button__link has-text-color no-border-radius" style="color:#ffffff">SHOP NOW</a></div>
<!-- /wp:button --></div>
<!-- /wp:buttons --></div></div>
<!-- /wp:cover --></div>
<!-- /wp:jetpack/layout-grid-column --></div>
<!-- /wp:jetpack/layout-grid -->
```

**Comment by ianstewart** [comment]
👍. I think the titles can be hard to read but I think it can work in context. (Like in the larger store layout you've put together.)

**Comment by alaczek** [comment]
I also have an issue with the text being smack dab in the middle of the face. Not sure this is a deal-breaker, as overall these are cool images, and I loathe to nitpick on images in general because I know how hard they are to find.

**Comment by pablohoneyhoney** [comment]
- Zooming in images can help.
- Larger labels and buttons can also increase legibility.

**Comment by iamtakashi** [comment]
![Pattern 119](https://user-images.githubusercontent.com/908665/93826044-50be5e00-fc5e-11ea-99c9-3d5504e32af0.jpg)

I've experimented with cropping to balance out the compositions. It feels better now.

> Larger labels and buttons can also increase legibility.

I made the label bigger and it helps. Button size is still something theme dependent, and I've opened a request for button control in the block level. https://github.com/WordPress/gutenberg/issues/20967
